mock/ecs_client.go

```diff
@@ -208,10 +206,10 @@ type ECSContainer struct {
 	TaskARN    *string
 	Name       *string
 	Image      *string
-	CPU        *int64
-	MemoryMB   *int64
-	Status     *string
-	GoalStatus *string
+	CPU        int32
```

Comment:
Same question as above.

Reply:
Made CPU also a pointer (because it's a pointer in [types.Container](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ecs@v1.27.1/types#Container))